ts/modules/accessibility/focusBorder.ts

```diff
@@ -73,9 +73,27 @@ extend(H.SVGElement.prototype, {
         bb.x += this.translateX ? this.translateX : 0;
         bb.y += this.translateY ? this.translateY : 0;
 
+        let borderPosX: number,
+            borderPosY: number;
+
+        // For text elements, apply x and y offset, #11397.
+        if (this.element.nodeName === 'text') {
+            const isRotated = Boolean(this.rotation);
+            const isFirefox = H.isFirefox;
+            borderPosX = this.attr('x') as number - (bb.width * 0.5) - pad +
```

Comment:
`as number` casting seems to avoidable - either through a check for `undefined` or parsing into a number. This depends on the TS error you were getting.